core/src/test/scala/com/salesforce/op/stages/impl/feature/StandardMinEstimatorTest.scala

```diff
@@ -0,0 +1,96 @@
+ *   and/or other materials provided with the distribution.
+ *
+ * * Neither the name of the copyright holder nor the names of its
+ *   contributors may be used to endorse or promote products derived from
+ *   this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package com.salesforce.op.stages.impl.feature
+
+import com.salesforce.op.OpWorkflow
+import com.salesforce.op.features.types._
+import com.salesforce.op.stages.base.unary.UnaryModel
+import com.salesforce.op.test.{OpEstimatorSpec, TestFeatureBuilder}
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+
+import scala.util.{Failure, Success}
+
+@RunWith(classOf[JUnitRunner])
+class StandardMinEstimatorTest extends OpEstimatorSpec[Real, UnaryModel[Real, Real], StandardMinEstimator[Real, Real]]{
+
+  val inputValues: Seq[Double] = Seq(10, 100, 1000)
+
+  /**
+   * Expected result of the transformer applied on the Input Dataset
+   */
+  override val expectedResult: Seq[Real] = {
+    val expectedMean = inputValues.sum / inputValues.length
+    val expectedStd = math.sqrt(inputValues.map(value => math.pow(expectedMean - value, 2)).sum
+      / (inputValues.length - 1))
+    val expectedMin = inputValues.min
+    inputValues.map( value => (value - expectedMin) / expectedStd).toReal
+  }
+
+  val (inputData, testF) = TestFeatureBuilder(inputValues.map(_.toReal))
+
+  override val estimator: StandardMinEstimator[Real, Real] = new StandardMinEstimator[Real, Real].setInput(testF)
+
+  it should "descale and work in standardized positive workflow" in {
+    val featureNormalizer = new StandardMinEstimator[Real, Real].setInput(testF)
+    val normedOutput = featureNormalizer.getOutput()
+    val metadata = featureNormalizer.fit(inputData).getMetadata()
+
+    val expectedMean = inputValues.sum / inputValues.length
+    val expectedStd = math.sqrt(inputValues.map(value => math.pow(expectedMean - value, 2)).sum
+      / (inputValues.length - 1))
+    val expectedMin = inputValues.min
+    val expectedSlope = 1 / expectedStd
+    val expectedIntercept = - expectedMin / expectedStd
+    ScalerMetadata(metadata) match {
```

Comment:
since we simply propagate `err`, maybe no need to do the pattern match here:
```
val meta = ScalerMetadata(metadata).get
meta shouldBe ...
...
```